tests/test_decorator_appender.py

```diff
@@ -1,26 +1,55 @@
 import pathlib
+import pytest
+from unittest import mock
 
+from fetching.projio import Project
 
-from tracing import TracerDecoratorAppender
+from fetching.strat import PyTestStrategy
 
-cwd = pathlib.Path.cwd() / 'resource' / "sample_decorator_appender_folder"
+cwd = pathlib.Path.cwd() / "tests" / "resource" / "sample_decorator_appender_folder"
 
 
-def test_if_test_object_searches_for_test_files_in_folders_it_generates_test_files():
-    test_object = TracerDecoratorAppender()
-    test_object.append_decorator_on_all_files_in(cwd, False)
-    test_file_paths = cwd.glob('test_*.py')
-    new_test_file_paths = [pathlib.Path(str(file_path).replace(".py", test_object.file_ending))
-                           for file_path in test_file_paths if not str(file_path).endswith(test_object.file_ending)]
+@pytest.fixture()
+def project_folder():
```

Comment:
Rename to get_project_mock.

Reply:
Mock naming conventions disallow this

Reply:
What is the standard mock naming convention then?

Reply:
The function name has to match the name of the parameters to test functions that use this mock.
i.e. the function has to be named like any other variable.